ml-algorithms/src/main/java/org/opensearch/ml/engine/utils/ZipUtils.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.engine.utils;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardCopyOption;
+import java.util.Enumeration;
+
+import org.apache.commons.compress.archivers.zip.ZipArchiveEntry;
+import org.apache.commons.compress.archivers.zip.ZipFile;
+import lombok.extern.log4j.Log4j2;
+
+/**
+ * A util class contains file related operations.
+ */
+@Log4j2
+public class ZipUtils {
+    public static void unzip(File zipFile, Path dest) {
+        try {
+            ZipFile file = new ZipFile(zipFile);
+            Enumeration<ZipArchiveEntry> en = file.getEntries();
+            ZipArchiveEntry ze;
+            while (en.hasMoreElements()) {
+                ze = en.nextElement();
+                String name = ze.getName();
+                if (name.contains("..")) {
```

Comment:
What does this mean? 

Do you have any test case covering this branch? Same question for line 35

Reply:
This is because previous djl unzip class wrote like this. I think it is for security concern but I cannot reproduce any file like that, so I cannot write a test case to cover it. Maybe delete it?